docs/coder_ping.md

```diff
@@ -0,0 +1,30 @@
+## coder ping
+
+Ping a Coder workspace
+
+```
+coder ping [workspace_name] [flags]
```

Comment:
what do you think about making this a subcommand, e.g. `coder ping workspace <workspace_name>` -- the reasoning being that we might want to be able to ping a replica or workspace provider or something else in the future? 

as it stands, there's no easy way to figure out where you should deploy things, except for deploying something first and then pinging.

we don't have to implement the other ping commands now, but making `coder ping workspace` a command means that we can implement them later

Reply:
Big fan!

Reply:
Going to move this under the existing `workspaces` command. `coder workspaces ping [name]` flows quite well!